packages/create-astro/test/git.test.js

```diff
@@ -12,21 +13,20 @@ describe('git', () => {
 		const context = { cwd: '', dryRun: true, prompt: () => ({ git: false }) };
 		await git(context);
 
-		expect(fixture.hasMessage('Skipping Git initialization')).to.be.true;
+		assert.ok(fixture.hasMessage('Skipping Git initialization'));
 	});
 
 	it('yes (--dry-run)', async () => {
-		const context = { cwd: '', dryRun: true, prompt: () => ({ git: true }) };
+		const context = { cwd: '', dryRun: true, prompt: () => ({ git: false }) };
```

Comment:
Why was this changed?

Reply:
Ohh, Seriously I also dont know 😅😅. I'll update it wait a minute. May be I'm trying some experiment but forgot to roll back.